llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2230,6 +2237,9 @@ Error Object::removeSections(
   for (auto &RemoveSec : make_range(Iter, std::end(Sections))) {
     for (auto &Segment : Segments)
       Segment->removeSection(RemoveSec.get());
+    if (isVerboseEnabled) {
+      llvm::outs() << "Removed Section: " << (RemoveSec.get()->Name);
+    }
```

Comment:
LLVM style doesn't use braces around single-line ifs, typically.
```suggestion
    if (isVerboseEnabled)
      llvm::outs() << "Removed Section: " << (RemoveSec.get()->Name);
```
Also, the GNU output uses lower case for its verbose output when printing which files are copied, so it makes sense to print it more like "removed section".